platform_versioned_docs/version-24.1/enterprise/_templates/k8s/data_studios/proxy.yml

```diff
@@ -19,13 +19,13 @@ spec:
         - name: proxy
           image: cr.seqera.io/private/nf-tower-enterprise/data-studio/connect-proxy:0.7.2
           command:
-            - caddy
             - serve
+            - caddy
           env:
             - name: CONNECT_HTTP_PORT
               value: "80"
             - name: CONNECT_REDIS_ADDRESS
-              value: <CONNECT_REDIS_ADDRESS>
+              value: <CONNECT_REDIS_ADDRESS>:6379
```

Comment:
suggestion: maybe change to <REDIS_ADDRESS_WITH_PORT>

Reply:
@endre-seqera, could, although it breaks the symmetry. All of these fields are `name: SOME_NAME` and then `value: <SOME_NAME>`. And here it is `CONNECT_REDIS_ADDRESS` rather than `REDIS_ADDRESS_WITH_PORT`.

Is it common to use a different Redis port?

Reply:
ok, it is not common to use different port 👍